Branch-SDK/src/main/java/io/branch/referral/Branch.java

```diff
@@ -735,6 +735,20 @@ public void setNetworkConnectTimeout(int connectTimeout) {
         }
     }
 
+    /**
+     * In cases of persistent no internet connection or offline modes,
+     * set a maximum number of attempts for the Branch Request to be tried.
+     *
+     * Must be greater than 0
+     * Defaults to 2
```

Comment:
@gdeluna-branch It says default here is 2, but then in prefHelper.java its `static final int DEFAULT_NO_CONNECTION_RETRY_MAX = 3;`

Reply:
Thanks for catching, fixed the comment to reflect.